src/games/supported/TimePilot.cpp

```diff
@@ -125,3 +128,10 @@ void TimePilotSettings::loadState(Deserializer & ser) {
   m_lives = ser.getInt();
 }
 
+DifficultyVect TimePilotSettings::getAvailableDifficulties(){
+    DifficultyVect diff;
+    diff.push_back(0);
```

Comment:
We need a comment here explaining why there are 3 (and not 4) settings.
(e.g. The manual says the valid settings are 1) L=B, R=B; 2) L=B, R=A; 3) L=A, R=B, with 3) being the hardest.)